### ISSUE TITLE
vue-loader@11.2.0 is crashing my app?

### ISSUE THREAD

**Issue by Pezhin** [vuejs/vue-loader#728]
I got this kind error in the new vue-loader11.2.0
```
Syntax Error: Unexpected token {


 @ ./~/vue-style-loader!./~/css-loader?{"minimize":false,"sourceMap":false}!./~/vue-loader/lib/style-compiler?{"id":"data-v-e0d15f88","scoped":false,"hasInlineConfig":false}!./~/vue-loader/lib/selector.js?type=styles&index=0!./src/App.vue 4:14-303 13:3-17:5 14:22-311
 @ ./src/App.vue
 @ ./src/main.js
 @ multi ./wwwroot/dev-client ./src/main.js

 error  in ./src/components/Hello.vue
```

which in "vue-loader@11.1.4" it works perfectly

**Comment by Pezhin** [comment]
More information:
It is crashing because I have 
```
<style scope>
#app {
  font-family: 'Avenir', Helvetica, Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  text-align: center;
  color: #2c3e50;
  margin-top: 60px;
}
</style>
``` 
in my .Vue files and if I delete those, it compiles

**Comment by cpylua** [comment]
They are using es6 features in published package, you may need to upgrade your node.

**Comment by h657070128** [comment]
I have the same problem with @Pezhin . I believe @cpylua is right. It is because of the version of node. I am using 5.5.0.

**Comment by Pezhin** [comment]
@h657070128: same here, I am using 5.5.0

@cpylua: so upgrading node is a must here?

**Comment by rsommer** [comment]
Upgrading to node7 fixed this problem for me.

**Comment by h657070128** [comment]
6.2.0 is enough in my case.

**Comment by hustcer** [comment]
+1

**Comment by RichardFans** [comment]
+1

**Comment by Daibai** [comment]
upgrading node

**Comment by FlyingIdeal** [comment]
thanks,The problem has been solved. you need update your node version

**Comment by jrenton** [comment]
There should not be a breaking change or a requirement to upgrade Node for a minor version.

**Comment by yyx990803** [comment]
I accidentally used ES2015 destructuring in some refactored code which is not supported in Node below 6. Fixed in 11.3.2.
